File: Latest_Build/talentpool/ui/js/reports/dataObj/CustomReportColumn.js
var CustomReportColumn = Class.create({
  initialize: function(key, displayName, groupBy, sortBy) {
	  this.key  = key;
	  this.displayName = displayName;
	  this.groupBy = groupBy;
	  this.sortBy= sortBy;
  },
  
    /*** Getters ***/
    getKey: function(){
		return this.key;
	},
	getDisplayName: function(){
		return this.displayName;
	},
	getGroupBy: function(){
		return this.groupBy;
	},
	getSortBy: function(){
		return this.sortBy;
	},
		
	/*** Setters ****/
	setKey: function(key){
		this.key=key;
	},
	setDisplayName: function(displayName){
		this.displayName=displayName;
	},
	setGroupBy: function(groupBy){
		this.groupBy=groupBy;
	},
	setSortBy: function(sortBy){
		this.sortBy=sortBy;
	}
});

File: src/com/talentPool/customReports/dataobject/CustomReportColumn.java
/**
 * 
 */
package com.talentPool.customReports.dataobject;

import java.io.Serializable;

/**
 * @author PraveenK
 * @since  Nov 23, 2011
 */
public class CustomReportColumn implements Serializable {
	
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	
	private String key;
	private String displayName; 
	private String groupBy;
	private String sortBy;
	
	/**
	 * @return the key
	 */
	public String getKey() {
		return key;
	}
	/**
	 * @param key the key to set
	 */
	public void setKey(String key) {
		this.key = key;
	}
	/**
	 * @return the displayName
	 */
	public String getDisplayName() {
		return displayName;
	}
	/**
	 * @param displayName the displayName to set
	 */
	public void setDisplayName(String displayName) {
		this.displayName = displayName;
	}
	/**
	 * @return the groupBy
	 */
	public String getGroupBy() {
		return groupBy;
	}
	/**
	 * @param groupBy the groupBy to set
	 */
	public void setGroupBy(String groupBy) {
		this.groupBy = groupBy;
	}
	/**
	 * @return the sortBy
	 */
	public String getSortBy() {
		return sortBy;
	}
	/**
	 * @param sortBy the sortBy to set
	 */
	public void setSortBy(String sortBy) {
		this.sortBy = sortBy;
	}

}
